trunk/Public/DynamicConditionsPublic.php

```diff
@@ -380,14 +401,16 @@ private function loopValues( $settings ) {
 
         // get value form conditions
         $compareType = self::checkEmpty( $settings, 'dynamicconditions_type', 'default' );
-        list( $checkValue, $checkValue2 ) = $this->getCheckValue( $compareType, $settings );
+        $checkValues = $this->getCheckValue( $compareType, $settings );
+        $checkValue = $checkValues[0];
+        $checkValue2 = $checkValues[1];
```

Comment:
What is the reasoning behind it to no longer use the `list` function?

Reply:
Because list have a different behavior in php7 and php5.

Reply:
According to the the [README.txt](https://github.com/RTO-Websites/dynamic-conditions/blob/9ac3170de802c52c6f32d2ae8d8be33317c11edb/trunk/README.txt#L6) this plugin requires PHP 7.0

Reply:
Because we don't explicitly test for Php5. But we shouldn't deliberately exclude users from Php5.
This only unnecessarily increases support requests.

Reply:
Okay was the usage of list previously made on the usage of php5? If yes how did the behavior change in Php7 to no longer use it?

Reply:
The order is upside down in Php5.

Reply:
So its a choice to provide php backwards compatibility got it. Any other comments about this @eac0n ?

Reply:
I think he doesnt have any other comments, because he has closed this conversation.